rero_ils/modules/loans/api.py

```diff
@@ -90,6 +90,16 @@ def patron_pid(self):
         """Shortcut for patron pid."""
         return self.get('patron_pid')
 
+    @property
+    def is_active(self):
```

Comment:
should be based on https://github.com/inveniosoftware/invenio-circulation/blob/f3b45538428a64ecc5ce3e2eebbc9767fb5aebd8/invenio_circulation/config.py#L35?